File: jja6312/2월 5일/최솟값 만들기.java
import java.io.*;
import java.util.*;

class Solution {
    public int solution(int []A, int []B) {
        int answer = 0;

        ArrayList<Integer> aArr = new ArrayList<>();
        ArrayList<Integer> bArr = new ArrayList<>();

        for(int i=0; i<A.length; i++){
            aArr.add(A[i]);
            bArr.add(B[i]);
        }

        Collections.sort(aArr);
        Collections.sort(bArr);

        for(int i=0; i<aArr.size(); i++){
            answer += aArr.get(i) * bArr.get(bArr.size()-1-i);
        }


        return answer;
    }
}

File: jeong-hyuk/2월5일/최솟값만들기.py
def solution(A, B):
    # A를 오름차순, B를 내림차순으로 정렬
    A.sort()
    B.sort(reverse=True)
    
    # 각 배열의 요소를 곱하여 더하기
    answer = sum(a * b for a, b in zip(A, B))
    
    return answer